examples/tutorials/05_ordinary_models.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<p><code>transpose</code> should be called as a namespace action <code>.transpose(B.images, ...)</code> or via <code>apply_transform_all</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>